extensions/data-transfer/portability-data-transfer-flickr/src/main/java/org/datatransferproject/datatransfer/flickr/photos/FlickrPhotosImporter.java

```diff
@@ -155,8 +155,12 @@ private void importSinglePhoto(UUID id, PhotoModel photo) throws FlickrException
       // with these (in case the album exists later).
       Preconditions.checkArgument(album != null, "Album not found: " + oldAlbumId);
 
-      Photoset photoset =
-          photosetsInterface.create(COPY_PREFIX + album.getName(), album.getDescription(), photoId);
+      // TODO: do we want to keep the COPY_PREFIX?  I feel like not
+      String albumName =
+          Strings.isNullOrEmpty(album.getName()) ? "" : COPY_PREFIX + album.getName();
```

Comment:
Does Flickr support importing an empty album name? or should we put in 'untitled'.  What happens if we get more than one empty album name?

Reply:
If there isn't a name in Google Photos, Google's API exports the album name as "Untitled".  There can be two albums with the same name in Flickr, and the photos are separated appropriately (no cross-contamination).